docs/requirements.txt

```diff
@@ -7,6 +7,8 @@ configargparse
 appdirs
 immutables
 sphinxawesome-theme
+snakemake
```

Comment:
```suggestion
```
Snakemake itself is imported from the repo checkout. It should not be needed and I would think it is even harmful here.